src/unstable-temp/DownloadImageModal/FormModel.tsx

```diff
@@ -173,6 +176,27 @@ const FormControl = ({ onModelChange, options, model }: FormControlProps) => {
 						</Box>
 					)}
 
+					{options.type === 'date' && (
+						<Input
+							mb={2}
+							type="date"
+							value={
+								!!model[options.name]
+									? moment(model[options.name] as string).format(DEFAULT_FORMAT)
+									: undefined
+							}
+							min={moment().utc().format(DEFAULT_FORMAT)}
```

Comment:
Why do we use UTC? I'm guessing the expiry date is either for the beginning of the day in UTC? If so, we should specify to the user that the expiry date is in UTC (and specify end time such as midnight), i.e. `Will expire at 11:59pm UTC on the selected date` so that there's no confusion when the key expires earlier or later than expected if a user thought it was in their local time

Reply:
Moved this to `datetime` with local time as discussed here: https://www.flowdock.com/app/rulemotion/resin-frontend/threads/pm6XnWFz9EyRW1QkLQ7OOq8OzBt